templates/header.opensuse.html

```diff
@@ -1,27 +1,10 @@
-{% extends "base.html" %}
+<!DOCTYPE html>
 
-{% block distribution_instruction %}
+<html lang="en">
 
-To use this repository, run the following command:
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="refresh" content="1; url='https://www.jenkins.io/doc/book/installing/linux/'" />
+</head>
 
-<pre class="text-white bg-dark">
-
-  sudo zypper addrepo -f {{web_url}}/ {{artifactName}}
```

Comment:
Where is the reference to `zypper` on https://www.jenkins.io/doc/book/installing/linux?

Reply:
That will need a larger addition to the installation section of the documentation.  There isn't a section for openSUSE installation yet.  I'll need to check with @kmartens27 to see if he's available to work on an openSUSE section for the documentation.